builder/docker/javascript/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM suborbital/subo:dev as subo
+
+FROM node:16-bullseye-slim
```

Comment:
I would have expected this to be the opposite... start with a `rust:latest` builder image to build Javy and then a second stage based on `node` that just copies the Javy binary.

Any reason why it's not?